sections/04_study.md

```diff
@@ -478,114 +478,103 @@ within neural networks.
 
 ### Protein secondary and tertiary structure
 
-Proteins play fundamental roles in all biological processes including the
-maintenance of cellular integrity, metabolism, transcription/translation, and
-cell-cell communication. Complete description of protein structures and
-functions is a fundamental step towards understanding biological life and
-also highly relevant to the development of therapeutics and drugs. UnitProt
-currently has about 94 millions of protein sequences. Even if we remove
-redundancy at 50% sequence identity level, UnitProt still has about
-20 millions of protein sequences. However, fewer than 100,000 proteins
-have experimentally-solved structures in Protein Data Bank (PDB). As a result,
-computational structure prediction is essential for a majority number of
-protein sequences. However, predicting protein 3D structures from sequence alone
-is very challenging, especially when similar solved structures (called templates)
-are not available in PDB. In the past decades, various computational methods have
-been developed to predict protein structure from different aspects,
-including prediction of secondary structure, torsion angles, solvent accessibility,
-inter-residue contact map, disorder regions and side-chain packing.
-
-Machine learning is extensively applied to predict protein structures and
-some success has been achieved. For example, secondary structure can be
-predicted with about 80% of 3-state (i.e. Q3) accuracy by a neural network
-method PSIPRED [@doi:10.1093/bioinformatics/16.4.404]. Starting from
-2012, deep learning has been gradually introduced to protein structure
-prediction. The adopted deep learning models include deep belief network,
-LSTM(long short-term memory), deep convolutional neural networks (DCNN)
-and deep convolutional neural fields[@doi:10.1007/978-3-319-46227-1_1
-@doi:10.1038/srep18962]. Here we focus on deep learning methods for
-two representative sub-problems: secondary structure prediction and
-contact map prediction. Secondary structure refers to local conformation of
```

Comment:
I think we should keep the "Secondary structure refers to..." sentence.  Many readers won't know these problems.